examples/google-cloud-speech/output/google-cloud-speech/lib/google/cloud/speech.rb

```diff
@@ -0,0 +1,125 @@
+
+module Google
+  module Cloud
+    ##
+    # # Ruby Client for Cloud Speech API
+    #
+    # [Cloud Speech API][Product Documentation]:
+    # Converts audio to text by applying powerful neural network models.
+    # - [Product Documentation][]
+    #
+    # ### Installation
+    #
+    # ```
+    # $ gem install google-cloud-speech
+    # ```
+    #
+    # ### Preview
+    #
+    # #### SpeechClient
+    #
+    # ```rb
+    # require "google/cloud/speech"
+    #
+    # speech_client = Google::Cloud::Speech.new
+    # language_code = "en-US"
+    # sample_rate_hertz = 44100
+    # encoding = :FLAC
+    # config = {
+    #   language_code: language_code,
+    #   sample_rate_hertz: sample_rate_hertz,
+    #   encoding: encoding
+    # }
+    # uri = "gs://bucket-name/hello.flac"
+    # audio = { uri: uri }
+    # response = speech_client.recognize(config, audio)
+    # ```
+    #
+    # ### Next Steps
+    #
+    # - Read the [Cloud Speech API Product documentation][Product Documentation]
+    #   to learn more about the product and see How-to Guides.
+    # - View this [repository's main README](https://github.com/googleapis/google-cloud-ruby/blob/master/README.md)
+    #   to see the full list of Cloud APIs that we cover.
+    #
+    # [Product Documentation]: https://cloud.google.com/speech
+    #
+    module Speech
+      FILE_DIR = File.realdirpath Pathname.new(__FILE__).join("..").join("speech")
+
+      AVAILABLE_VERSIONS = Dir["#{FILE_DIR}/*"].select { |file| File.directory?(file) }
+                                               .select { |dir| Google::Gax::VERSION_MATCHER.match(File.basename(dir)) }
+                                               .select { |dir| File.exist?(dir + ".rb") }
+                                               .map { |dir| File.basename(dir) }
+
+      ##
+      # Service that implements Google Cloud Speech API.
+      #
+      # @param version [Symbol, String]
+      #   The major version of the service to be used. By default :v1
+      #   is used.
+      # @overload new(version:, credentials:, scopes:, client_config:, timeout:)
+      #   @param credentials [Google::Auth::Credentials, String, Hash,
+      #     GRPC::Core::Channel, GRPC::Core::ChannelCredentials, Proc]
+      #     Provides the means for authenticating requests made by the client. This parameter can
+      #     be many types.
+      #     A `Google::Auth::Credentials` uses a the properties of its represented keyfile for
+      #     authenticating requests made by this client.
+      #     A `String` will be treated as the path to the keyfile to be used for the construction of
+      #     credentials for this client.
+      #     A `Hash` will be treated as the contents of a keyfile to be used for the construction of
+      #     credentials for this client.
+      #     A `GRPC::Core::Channel` will be used to make calls through.
+      #     A `GRPC::Core::ChannelCredentials` for the setting up the RPC client. The channel credentials
```

Comment:
I think the above two lines were meant to be a single sentence. Actually, I'm not sure this even makes sense. `GRPC::Core::Channel` and `GRPC::Core::ChannelCredentials` are both separately listed as allowed types, but this clause doesn't explain how.